src/components/ErrorBanner.js

```diff
@@ -19,55 +20,58 @@ class ErrorBanner extends Component {
     const onFirst = errorNumber === 0
     const onLast = errorNumber + 1 === errorState.length
 
-    return h(TopBanner, {
-      style: { padding: 0 },
-      isVisible: !!errorState.length,
-      onDismiss: () => {
-        this.setState({ errorNumber: 0 })
-        clearError()
-      }
-    },
-    [
-      div({
-        style: {
-          display: 'flex', alignItems: 'center', padding: '1rem', marginRight: '1rem',
-          borderRight: `1px solid ${colors.orange[2]}`
+    return createPortal(
```

Comment:
The only important change is `createPortal`. The rest of the changes are spacing/styling.